operations/research-and-development/product/product-management-team-handbook.md

```diff
@@ -277,7 +277,7 @@ Activities:
 * Learn about the product development process
   * High level overview \(PM buddy\)
     * Team structure
-    * [Development process overview](https://docs.mattermost.com/process/overview.html)
+    * [Product planning overview](https://handbook.mattermost.com/operations/research-and-development/product/product-planning/product-philosophy-and-principles)
```

Comment:
@wiersgallak, this was the closest approximation of the content that I could find, as the development process section isn't populated yet: https://handbook.mattermost.com/operations/research-and-development/product/development-process.